src/clientManager.js

```diff
@@ -140,9 +140,10 @@ ClientManager.prototype.removeUser = function(name) {
 	return true;
 };
 
-ClientManager.prototype.autoload = function(/* sockets */) {
+ClientManager.prototype.autoload = function ClientAutoLoad() {
 	var self = this;
-	setInterval(function() {
+
+	fs.watch(Helper.USERS_PATH, _.debounce(function ClientAutoLoadWatch() {
```

Comment:
This is just a callback, use the more concise arrow syntax instead `() => {`.
(Plus I'm really not a fan at all of starting a trend where function names are capitalized. This is good on class and class-like names but not callbacks.)